src/goto-programs/goto_program.cpp

```diff
@@ -776,6 +777,27 @@ void goto_programt::instructiont::validate(
       }
     };
 
+  // this option is always set to false until all source locations
+  // are reliably set correctly (future work)
+  if(goto_model_validation_options.check_source_location)
+  {
+    DATA_CHECK(
+      vm,
+      source_location.is_not_nil(),
+      "each instruction source location must not be nil");
+
+    DATA_CHECK(
+      vm,
+      code.source_location().is_not_nil(),
+      "each instruction code field, must have non nil source location");
+
```

Comment:
Remove the comma

Reply:
Note that not every instruction has a `code` member, so this will fail.

Reply:
In commit message (reproduced here in case it helps):

Removed previously disabled checks:
Not every instruction has a code member - so removed checks that both 
instruction sourcelocation and code sourcelocation are set and identical
Remove also remaining check that every instruction have a non-nil 
sourcelocation as this would have to be optional (if enabled fails 
many regression tests). This also simplifies considerably the overall 
validation pass (removes much passing around of the options structure).